File: Regionals/NA Pacific Northwest 2012/pacnw2012/PRACTICE/Practice.java
import java.util.Scanner;

public class Practice
{
	public static void main(String [] args)
	{
		Scanner kb = new Scanner(System.in);
		int n;

		n = kb.nextInt();
		kb.nextLine();//flush \n

		String line;

		for (int i = 0; i < n; i++)
		{
			line = kb.nextLine();
			System.out.println(line.charAt(line.length()-1));
		}

		
	}
}

File: Regionals/NA Pacific Northwest 2012/pacnw2012/PRACTICE/Practice.cpp
#include <iostream>

#include <string>
using namespace std;

int main(void)
{
	int n;
	
	string line;
	
	cin >> n;
	getline(cin, line);//flush \n
	
	for (int i = 1; i <= n; i++)
	{
		getline(cin, line);
		cout << line[line.size()-1] << endl;
	
	}
	
	return 0;
}